spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -122,21 +108,6 @@ object CometSparkSessionExtensions extends Logging {
     }
   }
 
-  private[comet] def isCometBroadCastForceEnabled(conf: SQLConf): Boolean = {
```

Comment:
We don't need a method to check a single config, especially when it is only called from one location. Replaced with a direct config check.